audit/projects/k8s-artifacts-prod-bak/buckets/asia.artifacts.k8s-artifacts-prod-bak.appspot.com/metadata.txt

```diff
@@ -14,5 +14,6 @@ gs://asia.artifacts.k8s-artifacts-prod-bak.appspot.com/ :
 	Time updated:			Thu, 09 Sep 2021 01:48:27 GMT
 	Metageneration:			34
 	Bucket Policy Only enabled:	True
+	Public access prevention:	unspecified
```

Comment:
Well this seems new

Reply:
Rollout of https://cloud.google.com/storage/docs/public-access-prevention ?